.travis.yml

```diff
@@ -7,22 +7,49 @@
 # Autobot is free software; you can redistribute it and/or modify it
 # under the terms of the MIT License; see LICENSE file for more details.
 
+notifications:
+  email: false
+
 language: python
-python:
-  - 3.7
-  - 3.6
-  - 3.5
 
-# Command to install dependencies, e.g. pip install -r requirements.txt --use-mirrors
-install: pip install -e .
+sudo: required
+
+dist: xenial
+
+python: 3.7
+
+cache:
```

Comment:
I think we usually do:

```yaml
cache:
  - pip
```

But if you can, double-check in Travis docs for Python builds